cluster/cluster.yaml

```diff
@@ -774,9 +768,6 @@ Resources:
         BlockPublicPolicy: true
         IgnorePublicAcls: true
         RestrictPublicBuckets: true
-      Tags:
-        - Key: application
-          Value: deployment-service
```

Comment:
Maybe we can have `component: deployment-service` at this level instead.

Reply:
Done.